scripts/retrieve_databundle_light.py

```diff
@@ -394,7 +463,11 @@ def download_and_unzip_post(config, rootpath, hot_run=True, disable_progress=Fal
         logger.info(f"Downloading resource '{resource}' from cloud '{url}'.")
 
         progress_retrieve(
-            url, file_path, data=postdata, disable_progress=disable_progress
+            url,
+            file_path,
+            data=postdata,
+            header=header,
+            disable_progress=disable_progress,
```

Comment:
Maybe this change is not needed, where is header defined?

Reply:
A header is added in the function loading hydrobasins. Removed it from here. Thanks for the catch!